docs/src/main/paradox/server-side/server-https-support.md

```diff
@@ -46,23 +46,10 @@ Scala
 Java
 :  @@snip [SimpleServerApp.java](/akka-http-tests/src/main/java/akka/http/javadsl/server/examples/simple/SimpleServerApp.java) { #https-http-config }
 
-@scala[Once you configured the HTTPS context, you can set it as default:]
-@java[Then pass it to the `akka.http.javadsl.Http` class's `setDefaultServerHttpContext` method, like in the below `main` method.]
-
-Scala
-:  @@snip [HttpsServerExampleSpec.scala]($test$/scala/docs/http/scaladsl/server/HttpsServerExampleSpec.scala) { #set-low-level-context-default }
-
-Java
-: @@snip [SimpleServerApp.java](/akka-http-tests/src/main/java/akka/http/javadsl/server/examples/simple/SimpleServerApp.java) { #https-http-app }
```

Comment:
There's code somewhere with a `// #https-http-app` comment that we can probably remove (or at least remove the comment ;-) )

Reply:
Looks like https://github.com/akka/akka-http/blob/ea534cca2cb0a99e7121df92adfd2302291c224e/akka-http-tests/src/main/java/akka/http/javadsl/server/examples/simple/SimpleServerApp.java#L44-L129